test/common/buffer/buffer_fuzz.cc

```diff
@@ -382,14 +378,14 @@ uint32_t bufferAction(Context& ctxt, char insert_value, uint32_t max_alloc, Buff
     break;
   }
   case test::common::buffer::Action::kGetRawSlices: {
-    const uint64_t slices_needed = target_buffer.getRawSlices(nullptr, 0);
+    const uint64_t slices_needed = target_buffer.getRawSlices().size();
     const uint64_t slices_tested =
         std::min(slices_needed, static_cast<uint64_t>(action.get_raw_slices()));
     if (slices_tested == 0) {
       break;
     }
-    std::vector<Buffer::RawSlice> raw_slices{slices_tested};
-    const uint64_t slices_obtained = target_buffer.getRawSlices(raw_slices.data(), slices_tested);
+    Buffer::RawSliceVector raw_slices = target_buffer.getRawSlices(/*max_slices=*/slices_tested);
```

Comment:
We don't usually put the /*max_slices=*/ comments in Envoy; could be addressed in follow-up.

I did notice there are some existing ones in the quic code though.

I personally don't care, but just pointing that out, and in this case I think it's reasonably intuitive what it means.